src/main/java/com/google/devtools/build/lib/remote/SimpleBlobStoreFactory.java

```diff
@@ -35,8 +36,13 @@ private SimpleBlobStoreFactory() {}
 
   public static SimpleBlobStore createRest(RemoteOptions options, Credentials creds)
       throws IOException {
-    return new RestBlobStore(
-        options.remoteHttpCache, (int) TimeUnit.SECONDS.toMillis(options.remoteTimeout), creds);
+    try {
+      return new HttpBlobStore(
+          URI.create(options.remoteHttpCache), (int) TimeUnit.SECONDS.toMillis(options.remoteTimeout),
```

Comment:
Prefer passing in Duration instead of int, and maybe make options.remoteTimeout a Duration field. We have a converter to Duration which accepts time suffixes (12s, 12h, etc.).